client/src/components/JobList.vue

```diff
@@ -60,6 +69,7 @@
           const accessToken = authenticationService.getAccessToken();
           jobsApi.fetchAll(accessToken)
             .then((jobs) => {
+              this.allJobs = this._sortJobsByProjectStatus(jobs);
               this.jobs = this._sortJobsByProjectStatus(jobs);
```

Comment:
Not really happy with these two duplicated lines. Could have used a deepClone maybe. Taking suggestions :)

Reply:
allJobs and filteredJobs ?

Reply:
j'ai choisi displayedJobs